adoc/chapters/programming_interface.adoc

```diff
@@ -6438,6 +6452,15 @@ being bound to a <<command-group>> with [code]#handler::require()#, the
 implementation throws a synchronous [code]#exception# with the
 [code]#errc::kernel_argument# error code when the <<command>> is submitted.
 
+An [code]#accessor# to a buffer is only a view over that buffer.  Therefore,
+it is the application's responsibility to ensure that the lifetime of the
+buffer is at least as long as the lifetime of the accessor.  Calling any
+member function on an accessor after its associated buffer has been destroyed
+results in undefined behavior.  In some cases the buffer destructor is defined
+to block until commands with requisites on that buffer have completed.  This
+results in well defined behavior so long as all invocations to member functions
+of accessors for that buffer occur before those commands complete.
```

Comment:
I have the feeling that the buffer constructor should block if there are accessors using it or at least being internally kept alive if there is no side effect on buffer use or buffer destruction.

Reply:
I can give up on this.